src/js/edu-benefits/components/military-history/RotcScholarship.jsx

```diff
@@ -9,25 +9,29 @@ export default class RotcScholarship extends React.Component {
     const { view, onValueChange } = this.props;
     const scholarship = this.props.data;
     const formFields = (
-      <div>
-        <ErrorableTextInput
-            errorMessage={validateIfDirty(scholarship.amount, isNotBlank) ? undefined : 'Please enter the amount received.'}
-            validation={validateIfDirty(scholarship.amount, isNotBlank)}
-            label="Amount recieved"
-            name="amount"
-            field={scholarship.amount}
-            placeholder="$"
-            required
-            onValueChange={(update) => {onValueChange('amount', update);}}/>
-        <ErrorableTextInput
-            errorMessage={validateIfDirty(scholarship.year, isValidYear) ? undefined : 'Please enter a valid year.'}
-            validation={validateIfDirty(scholarship.year, isValidYear)}
-            label="Year recieved"
-            name="year"
-            field={scholarship.year}
-            placeholder="yyyy"
-            required
-            onValueChange={(update) => {onValueChange('year', update);}}/>
+      <div className="row">
+        <div className="columns small-4">
+          <ErrorableTextInput
+              errorMessage={validateIfDirty(scholarship.year, isValidYear) ? undefined : 'Please enter a valid year.'}
+              validation={validateIfDirty(scholarship.year, isValidYear)}
+              label="Year"
+              name="year"
+              field={scholarship.year}
+              placeholder="yyyy"
+              required
+              onValueChange={(update) => {onValueChange('year', update);}}/>
+        </div>
+        <div className="columns small-8">
+          <ErrorableTextInput
+              errorMessage={validateIfDirty(scholarship.amount, isNotBlank) ? undefined : 'Please enter the amount received.'}
+              validation={validateIfDirty(scholarship.amount, isNotBlank)}
+              label="Scholarship amount"
+              name="amount"
+              field={scholarship.amount}
+              placeholder="$"
```

Comment:
Do you think people will interpret this as needing a `$` in front of their input? Like entering `$10` instead of just `10`? I'm assuming this is not required.